src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppAppRunner.recipe

```diff
@@ -412,14 +412,29 @@
             "AdvancedSetting": true,
             "Updatable": true,
             "ChildOptionSettings": [
+                {
```

Comment:
Adding this is potentially a breaking change for existing users. For example in current deployments if a user selected create a new VPC connector. Then they take this update. Unless the change this setting to true during redeployments they will lose the VPC connector.

I think we have to make this change under the guise of the tool being in preview but we should call this out in the changelog.